os-server/src/main/java/app/openschool/course/discussion/DiscussionForumController.java

```diff
@@ -301,11 +304,12 @@ public ResponseEntity<MentorQuestionResponseDto> createMentorQuestion(
   @GetMapping("/{enrolledCourseId}/mentor-questions")
   public ResponseEntity<Page<MentorQuestionResponseDto>> findMentorQuestionsByCourseId(
       @Parameter(description = "Enrolled course id") @PathVariable Long enrolledCourseId,
-      Pageable pageable) {
+      Pageable pageable,
+      @Valid @Size(min = 3) @RequestParam(value = "q", required = false) String searchQuery) {
```

Comment:
please define a custom validation message for this case

Reply:
Done✅